src/runtime/CMakeLists.txt

```diff
@@ -60,6 +60,9 @@ set(INCLUDE_DIRS
 
 add_library(hipSYCL-rt SHARED ${HIPSYCL_SOURCES})
 target_include_directories(hipSYCL-rt PRIVATE ${INCLUDE_DIRS})
+# syclcc already knows about these include directories, but clangd-based tooling does not. Specifying them
+# explicitly ensures that IDEs can resolve all hipSYCL includes correctly.
+target_include_directories(hipSYCL-rt INTERFACE "$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>")
```

Comment:
Also add `target_include_directories(hipSYCL-rt INTERFACE "$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include/hipSYCL/contrib>")` so that `"hipCPU/hip/hip_runtime.h"` is found and and integration with IDE fully works